daemon/cmd/kube_proxy_replacement.go

```diff
@@ -712,6 +714,30 @@ func detectNodeDevice(ifidxByAddr map[string]int) (string, error) {
 	return link.Attrs().Name, nil
 }
 
+// expandDevices expands all wildcard device names to concrete devices.
+// e.g. device "eth+" expands to "eth0,eth1" etc. Non-matching wildcards are ignored.
+func expandDevices() {
+	allLinks, err := netlink.LinkList()
+	if err != nil {
+		log.WithError(err).Fatal("Cannot list network devices via netlink")
+	}
```

Comment:
I'm not sure how "bad" `netlink.LinkList()` is by doing this on agent's initialization. Assuming that it is "slow" and has a bad memory consumption, on a node that has 100 veth pairs, could we run the `netlink.LinkList()` only if any of the `option.Config.Devices` had a suffix `+`? :pray: If it is not that bad then feel free to ignore my comment.

Reply:
Ah good point. I'll test out the performance with say ~1k dummies and if it's acceptable I'll leave it, otherwise will change it.

Reply:
Ok, here's stats:
10: 250us, 52kB runtime.MemStats.TotalAlloc increase
100: 1.6ms, 470kB *-*
1000: 11ms, 4.4MB *-* 

This seems reasonable enough to me. @aanm agreed?

Reply:
yep, sounds reasonable to me as well. Thanks for looking!